osgp/platform/osgp-adapter-domain-smartmetering/src/main/java/org/opensmartgridplatform/adapter/domain/smartmetering/application/services/FirmwareService.java

```diff
@@ -260,46 +259,6 @@ private void checkFirmwareModuleTypes(final Set<FirmwareModuleType> firmwareModu
         deviceModel, moduleDescription, moduleVersion);
   }
 
-  public SmartMeter storeFirmware(
```

Comment:
Storing FirmwareFiles is no longer needed for a device?

Reply:
Correct, this is historic data, it is not needed anymore!